scripts/generator.ts

```diff
@@ -295,13 +324,18 @@ function generateReactComponent({ name, js }: SchemaHTMLElement, { packageName,
 
   const elementName = convertElementNameToClassName(name);
   const elementModulePath = createImportPath(relative(nodeModulesDir, path), false);
+  const themePath = elementsWithoutTheme.some((el) => path.includes(el))
+    ? path
+    : path.replace('/vaadin-', '/theme/lumo/vaadin-');
```

Comment:
Importing from Lumo theme should no longer be necessary.

Reply:
Removed.